### ISSUE TITLE
Project Detail route should use except name

### ISSUE THREAD

**Issue by fozbek** [metinorak/devSeater#24]
When i change the project name and refresh the page, I see 404.

**Comment by metinorak** [comment]
Yes, you're right. Project page link is changing because of project name is included in url. Maybe project manager should be warned about this changing. And updating project's name should not be easy like this. A password confirmation can be requested by the manager.

**Comment by fozbek** [comment]
also when i dont change the value, i get 'bla bla updated' message. i think you can use **change** event instead of **focusout**.

**Comment by metinorak** [comment]
Yes, change event is more appropriate for this action.
